sources/world/world_countries_v7.hjson

```diff
@@ -0,0 +1,235 @@
+{
+  versions: '>=7'
+  production: true
+  type: http
+  note: Administrative map units from Natural Earth Data. URLs listed at http://geojson.xyz
+  wikidata: Q6256
+  data: ''
+  attribution: [
+    {
+      label: {
+        en: Made with NaturalEarth
+      }
+      url: {
+        en: http://www.naturalearthdata.com/about/terms-of-use
+      }
+    }
+    {
+      label: {
+        en: World Bank
+      }
+      url: {
+        en: https://data.worldbank.org/
+      }
+    }
+    {
+      label: {
+        en: Wikidata
+      }
+      url: {
+        en: https://www.wikidata.org/
+      }
+    }
+    {
+      label: {
+        en: Elastic Maps Service
+      }
+      url: {
+        en: https://www.elastic.co/elastic-maps-service
+      }
+    }
+  ]
+  fieldMapping: [
+    {
+      type: id
+      name: iso2
+      desc: Two letter abbreviation
+    }
+    {
+      type: id
+      name: iso3
+      desc: Three letter abbreviation
+    }
+    {
+      type: property
+      name: name
+      desc: Country name
+    }
+    {
+      type: property
```

Comment:
Same as above
> I think we're introducing a new data type here. I suggest metric. Otherwise, population and area will show up as join fields in Elastic Maps.